src/Components/RosTable/RosTable.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Table, TableHeader, TableBody, expandable } from '@patternfly/react-table';
+import propTypes from 'prop-types';
+import { flatMap } from 'lodash';
+import { ExpandedRow } from './ExpandedRow';
+import { ProgressScoreBar } from './ProgressScoreBar';
+import { EmptyTable } from '@redhat-cloud-services/frontend-components';
+import { EmptyStateDisplay } from '../EmptyStateDisplay/EmptyStateDisplay';
+import './RosTable.scss';
+
+const actionLink = (id, textValue, classAsPerType, linkPath) => (
+    <a href={ linkPath } className={ `pf-link ${classAsPerType} link-${id}` }>{textValue}</a>
+);
+
+const renderSystemLink = (id, textValue) => {
```

Comment:
Instead of this function I'd use directly the `actionLink` function.

Reply:
makes sense :+1: